test_schemas/en/test_interstitial_instruction.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
Like Validator a little bit verbose, as we could just have the interstitial and a confirmation block, but nothing major